test/test_tensorexpr.py

```diff
@@ -1065,6 +1066,41 @@ def easy(x, y):
         # FIXME: interp.elapsed_value() also increments due to simplifier
         assert llvm.elapsed_value() == 1 or interp.elapsed_value() > 1
 
+    def _test_softmax(self, device):
+        @torch.jit.script
+        def test(x, y):
+            a = F.softmax(x, dim=0)
+            b = F.softmax(y, dim=0)
+            c = F.softmax(x, dim=1)
+            d = F.softmax(y, dim=1)
+            return a + b + c + d
+
+        def test_ref(x, y):
+            a = special.softmax(x, axis=0)
+            b = special.softmax(y, axis=0)
+            c = special.softmax(x, axis=1)
+            d = special.softmax(y, axis=1)
+            return a + b + c + d
+
+        traced = torch.jit.trace(test, (torch.randn(2,3), torch.rand(2,3)))
+        inp = torch.randn(2, 3, device=device)
+        res = traced(inp, inp)
+        ref = traced(inp, inp)
```

Comment:
`res` and `ref` seem to be produced in the same way. Did you mean to call `test_ref`?

Reply:
Fixed.